.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,9 @@
 https://github.com/WordPress/gutenberg/blob/trunk/CONTRIBUTING.md -->
 
 ## What?
+<!-- Link this PR with the issue it addresses -->
```

Comment:
It should be good to specify in the instructions that linking to an issue is encouraged, although not strictly necessary. There can also be cases where the PR doesn't close an issue but is a "Follow up to #XXXX".

Reply:
Will this work - https://github.com/WordPress/gutenberg/pull/68924/commits/0f32309870c58f5e45664a866c0fb88761d21eff?